index.js

```diff
@@ -6,7 +6,7 @@ module.exports = ({ url, iosManifest, config }) => {
     slack.send(
       {
         icon_url: iosManifest.iconUrl,
-        text: `${iosManifest.name} v${iosManifest.version} published to ${url}`,
+        text: `${iosManifest.name} v${iosManifest.version} published to ${url}. Channel=${iosManifest.releaseChannel || 'default'}`,
```

Comment:
@brentvatne maybe it would be better to put the release channel in url with a query string no?

Reply:
yeah i think that makes sense. if it's default, we can just leave it out entirely